libvmaf/src/dict.c

```diff
@@ -6,7 +6,7 @@
 #include "dict.h"
 #include "libvmaf/feature.h"
 
-const VmafDictionaryEntry *vmaf_dictionary_get(VmafDictionary **dict,
+VmafDictionaryEntry *vmaf_dictionary_get(VmafDictionary **dict,
                                                const char *key, uint64_t flags)
```

Comment:
Might want to reformat these two lines, I don't know if @kylophone used a clang-format, so you might need to just manually delete the spaces on line 10 to match the open paren